src/components/settings/SettingsLMSTab/ErrorReporting/ContentMetadataTable.jsx

```diff
@@ -9,38 +11,81 @@ import DownloadCsvButton from './DownloadCsvButton';
 import { getSyncStatus, getSyncTime } from './utils';
 
 function ContentMetadataTable({ config, enterpriseCustomerUuid }) {
-  const [data, setData] = useState([]);
+  const [currentPage, setCurrentPage] = useState();
+  const [currentFilters, setCurrentFilters] = useState();
+  const [paginationData, setPaginationData] = useState({
+    itemCount: 0,
+    pageCount: 0,
+    data: [],
+    sortBy: '',
+  });
 
   useEffect(() => {
     const fetchData = async () => {
       const response = await LmsApiService.fetchContentMetadataItemTransmission(
-        enterpriseCustomerUuid, config.channelCode, config.id,
+        enterpriseCustomerUuid, config.channelCode, config.id, currentPage, currentFilters,
       );
       return response;
     };
 
     fetchData()
       .then((response) => {
-        setData(response.data.results);
+        setPaginationData({
+          itemCount: response.data.count,
+          pageCount: response.data.pages_count,
+          data: response.data.results,
+        });
       })
       .catch((err) => {
         logError(err);
       });
-  }, [config.channelCode, config.id, enterpriseCustomerUuid]);
+  }, [config.channelCode, config.id, enterpriseCustomerUuid, currentPage, currentFilters]);
+
+  // Call back function, handles filters and page changes
+  const fetchData = useCallback(
+    (args) => {
+      const newFiltersMap = args.filters.map((filter) => ({ [filter.id]: filter.value }));
+      const newFilters = Object.assign({}, ...newFiltersMap);
+
+      const newSortByMap = args.sortBy.map((filter) => ({ sort_by: `${filter.desc ? '-' : ''}${filter.id}` }));
+      const newSortBys = Object.assign(newFilters, ...newSortByMap);
```

Comment:
Not necessary to expand the filter queries if we're only using the last one, and don't need to redundantly roll in the filters if we're already combining them below.
```suggestion
      const sort_by = args.sortBy.at(-1);
      const newSortBys = { sort_by: `${sort_by.desc ? '-' : ''}${sort_by.id}` };
```